cli.js

```diff
@@ -59,6 +61,31 @@ program
       // For perf, don't delete the repository.  This means users will have to
       // add .gh-pages-cache to their .gitignore file to hide its `git status`.
       // return rimraf('.gh-pages-cache');
+
+      fs.access('.gitignore').then(function() {
+        var gitignore = fs.readFileSync('.gitignore', 'utf8');
+
+        if (gitignore.indexOf('.gh-pages.cache') === -1) {
+          promptly.prompt('.gh-pages-cache is a temporary repository that we use to push changes to your gh-pages branch. Do you want to add it to .gitignore (Y/N, suggested Y): ', {
```

Comment:
Nit: *promptly.confirm* handles y/n prompts and does the validation for you, which would simplify this implementation.